lib/taskbook.js

```diff
@@ -499,6 +499,18 @@ class Taskbook {
     this._save(_data);
     render.successPriority(id, level);
   }
+
+  clean() {
+    const keysToDelete = [];
```

Comment:
To make things simpler `keysToDelete` can be renamed to just `ids` and `_data` can be explicitly declared & pushed to the top:

```js
const ids = [];
const {_data} = this;
```